.travis.yml

```diff
@@ -40,14 +40,17 @@ addons:
 env:
   global:
     - ES2_DOWNLOAD_URL="https://download.elastic.co/elasticsearch/release/org/elasticsearch/distribution/tar/elasticsearch/2.4.5/elasticsearch-2.4.5.tar.gz"
+    - ES7_DOWNLOAD_URL="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.0.1-linux-x86_64.tar.gz"
     - ES_HOST=127.0.0.1
   matrix:
     - REQUIREMENTS=lowest ES_URL=$ES2_DOWNLOAD_URL BROKER_URL=redis://localhost:6379/0
```

Comment:
Don't forget to remove ES2 to ligthen the Travis builds as well for others. We can discuss what should stay/go

Reply:
We did it here: https://github.com/inveniosoftware/invenio-stats/pull/85